west.yml

```diff
@@ -142,7 +142,7 @@ manifest:
       groups:
         - fs
     - name: hal_adi
-      revision: a9ca7d172152789a02439bbdfb94feb72e30553d
+      revision: pull/16/head
```

Comment:
zephyrproject-rtos/hal_adi#16 has been merged, so this can be updated to the new SHA